lib/stdio/generator.js

```diff
@@ -0,0 +1,100 @@
+import {Duplex, Readable, PassThrough, getDefaultHighWaterMark} from 'node:stream';
+import {callbackify} from 'node:util';
+import {setTimeout} from 'node:timers/promises';
+
+/*
+ * Generators can be used to transform/filter standard streams.
```

Comment:
Without the `*` prefixes.

Reply:
Fixed in 295f63a6922061e72db981cac2dacc3ff177b05c